src/util/operations.spec.tsx

```diff
@@ -44,3 +44,21 @@ describe("getInstanceName", () => {
     expect(name).toBe("testInstance3");
   });
 });
+
+describe("getProjectName", () => {
+  it("identifies project name from an instance operation when no project parameter is present", () => {
+    const operation = craftOperation("/1.0/instances/testInstance1");
+    const name = getProjectName(operation);
+
+    expect(name).toBe("default");
+  });
+
+  it("identifies project name from an instance operation in a custom project", () => {
+    const operation = craftOperation(
+      "/1.0/instances/testInstance2?project=project"
+    );
+    const name = getProjectName(operation);
+
+    expect(name).toBe("project");
+  });
```

Comment:
Please add a third test for when project is not the first parameter, something like `/1.0/instances/testInstance1?foo=bar&project=barProject`. Looking at the implementation, this will likely fail, please fix the implementation for this case.

Reply:
Updated the implementation